build/sdk.sh

```diff
@@ -34,7 +34,6 @@ SCHEMES_TO_BUILD = [
   "NoteSync",
   "OCMock",
   "OCMockLib",
-  "React",
```

Comment:
It is there line 42.
The react scheme doesn't exist. (We don't compile the React project from the SDK workspace).